Server/PrivateHeaders/CDStructures.h

```diff
@@ -11,8 +11,6 @@
 
 typedef void (^CDUnknownBlockType)(void); // return type and parameters are unknown
 
-typedef void (^CDUnknownFunctionPointerType)(void); // return type and parameters are unknown
```

Comment:
why removed? no alternative? why using `CDUnknownBlockType` instead is a correct approach?